src/aiortc/rtcrtpsender.py

```diff
@@ -201,11 +226,19 @@ async def stop(self):
             await asyncio.gather(self.__rtp_exited.wait(), self.__rtcp_exited.wait())
 
     async def _handle_rtcp_packet(self, packet):
+        self.__log_debug("< RTCP %s arrival time: %s",
+                packet, datetime.datetime.now())
+        
         if isinstance(packet, (RtcpRrPacket, RtcpSrPacket)):
             for report in filter(lambda x: x.ssrc == self._ssrc, packet.reports):
                 # estimate round-trip time
-                if self.__lsr == report.lsr and report.dlsr:
-                    rtt = time.time() - self.__lsr_time - (report.dlsr / 65536)
+                #if self.__lsr == report.lsr and report.dlsr:
+                if report.lsr in self.__lsr_list and report.dlsr:
```

Comment:
how much of this is being used/is useful? would help to clean up this file and remove some things that aren't necessary

Reply:
removed all the code that is not completed